index.js

```diff
@@ -15,16 +15,16 @@ module.exports = fp(function (fastify, options, next) {
 
     // static salt to be used for key derivation, not great for security,
     // but better than nothing
-    var salt = 'mq9hDxBVDbspDR6nLfFT1g=='
+    var salt = 'mq9hDxBVDbspDR6n'
```

Comment:
I would prefer the default salt __not__ be changed. In other terms store it as a buffer.

Reply:
The previous salt was 24 character.When It converted  to Buffer by `Buffer.from(salt, 'base64')` the length of result buffer is 16 Byte. So I changed the length of default salt to 16 character  and for converting  it to buffer I've used `Buffer.from(salt, 'ascii')`

Reply:
You are altering the actual bytes in the Buffer resulting the parsed operation. Truncating the string is not enough.

Reply:
I thought that this string is a random string. How can I generate it.

Reply:
It was a random string. However it's our current default, and changing it is an unneeded semver-major change.

You can do: 

```js
const defaultSalt = Buffer.from('mq9hDxBVDbspDR6nLfFT1g==', 'base64')
```

and then pass it down as a buffer.